app/controllers/aws_parameter_store_controller.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+class AwsParameterStoreController
+
+  def initialize
+    @current_file = File.basename(__FILE__)
+    @systems_manager = create_client
+  end
+
+  # Call AWS parameter store to get google-sheet credential values.
+  def get_parameter_store_values(parameter_name)
+    begin
+      response = @systems_manager.get_parameter({
+        name: parameter_name,
+        with_decryption: false
+      })
+      LogWrapper.log('INFO', {'message' => "AWS parameter call  #{response}",
+                              'method' => 'get_parameter_store_values'})
+      response_body = response.parameter
+    rescue Aws::Errors::ServiceError => e
+      raise e
+    end
+    response_body
+  end
+
+
+  def news_letter_google_sheet_credentials
+    get_parameter_store_values("/#{ENV[RAILS_ENV]}/google-credentials/mylibrarynyc")
+  end
+
+
+  def google_sheet_credentials
+    client_secret_body = news_letter_google_sheet_credentials
+    JSON.parse client_secret_body.value if client_secret_body.present?
```

Comment:
does there need to be a try-catch around the parsing, or is there one later up in the code?